apps/nextra/pages/en/build/smart-contracts.mdx

```diff
@@ -39,6 +43,10 @@ Move allows developers to write programs that flexibly manage and transfer asset
     <Card.Title linkType="external">Aptos Move Examples</Card.Title>
     <Card.Description>30+ examples on how to develop Move on Aptos</Card.Description>
   </Card>
+  <Card href="https://github.com/aptos-labs/aptos-core/tree/main/aptos-move/move-examples">
+    <Card.Title>Aptos Move by Example →</Card.Title>
+    <Card.Description>Explanations of Move concepts and 10+ examples including auctions, lotteries, voting, and more, curated by Move Developers DAO</Card.Description>
+  </Card>
```

Comment:
Most of these are on aptos learn?

These are also not curated by Move developers dao

Reply:
Changed